dockerfiles/alpine/is/Dockerfile

```
@@ -70,6 +70,7 @@ ARG WSO2_SERVER_VERSION=6.0.0
 ARG WSO2_SERVER_REPOSITORY=product-is
 ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+ARG WSO2_SERVER_DIST_URL
```

Add a comment on top of this. Fix in other places too
```suggestion
ARG WSO2_SERVER_DIST_URL = ""
```

---

Keep a empty value assigned